website/docs/r/catalog.html.markdown

```diff
@@ -0,0 +1,36 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_catalog"
+sidebar_current: "docs-vcd-resource-catalog"
+description: |-
+Provides a vCloud Director catalog resource. This can be used to create and delete catalog.
+---
+
+# vcd\_catalog
+
+Provides a vCloud Director catalog resource. This can be used to create and delete catalog.
+
+Supported in provider *v2.0+*
+
+## Example Usage
+
+```
+resource "vcd_catalog" "myNewCatalog" {
+  org = "my-org"
+
+  name = "my-catalog"
+  description = "catalog for files"
+  delete_force      = "true"
+  delete_recursive  = "true"  
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `org` - (Optional) The name of organization to use. Not needed if defined at provider level
+* `name` - (Required) Catalog name
+* `description` - (Optional) - Description of catalog
+* `delete_force` - (Required) - When destroying use delete_force=True with delete_recursive=True to remove a catalog and any objects it contains, regardless of their state.
```

Comment:
Please move both `delete_force` write-up and the parameter in the example above under the `delete_recursive` parameter. Then it will be enough for the new user to read the section once (as opposed to twice) to understand how it works :)

Reply:
Looks great now, thanks!